giscus.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We should delete this file, and add it as an exclusion to the merge script.

Reply:
It is an interesting feature, but it isn't supported at all here.

Reply:
done.